src/docs/perf/rendering/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Improving rendering performance
+description: How to measure and evaluate your app's rendering performance.
+---
+
+Rendering animations in your app is one of the most cited
+topics of interest when it comes to measuring performance. 
+Thanks, in part, to Flutter's Skia engine and its ability
+to quickly create and dispose of widgets, 
+Flutter applications are performant by default,
+so you only need to avoid common pitfalls to achieve
```

Comment:
common pitfalls:
* rebuilding far more of the app than expected each frame (link to widget rebuild tracker feature in IntelliJ).
* Building a large list of children directly instead of using a ListView
* See https://github.com/InMatrix/perf_diagnosis_demo/tree/master/lib
  for more examples